src/map/clif.c

```diff
@@ -22215,6 +22215,104 @@ static bool clif_enchant_equipment(struct map_session_data *sd, enum equip_pos p
 #endif
 }
 
+static bool clif_lapineDdukDdak_open(struct map_session_data *sd, int itemId)
+{
+#if PACKETVER_MAIN_NUM >= 20160601 || PACKETVER_RE_NUM >= 20160525 || defined(PACKETVER_ZERO)
```

Comment:
instead of this long check, which can be break easily later if needed to change how about do something like
```
#if PACKETVER_MAIN_NUM >= 20160601 || PACKETVER_RE_NUM >= 20160525 || defined(PACKETVER_ZERO)
#define LAPINDDUKDDAK_ENABLED
#endif 
```
and change all to just check ```#ifdef LAPINDDUKDDAK_ENABLED```

Reply:
this add overhead and will be harder to check what versions really enabled here
each packet may have own allowed versions...